docs/guides/1230-persistent-cache.md

```diff
@@ -0,0 +1,216 @@
+[✔] client.filesystem.".".read                                              0.1s
+[✔] actions.build.container.export                                          0.1s
+[✔] client.filesystem."./output".write                                      0.2s
+```
+
+Indeed, Dagger has an ephemeral cache so if you rerun it, that shouldn't take
+that long.
+
+```shell
+[✔] actions.build.container                                                 2.9s
+[✔] client.filesystem.".".read                                              0.0s
+[✔] actions.build.container.export                                          0.0s
+[✔] client.filesystem."./output".write                                      0.1s
+```
+
+But if you stop Buildkit daemon and remove volume, cache will be lost and
+all actions will be executed again.
+
+:::info
+Now we have seen how ephemeral cache works, let's continue to understand how
+store cache in your local filesystem, so you can clean your docker engine without
+losing all your CI's cache.
+:::
+
+## Persistent cache in your local filesystem
+
+To store cache in your local filesystem, you don't need much effort : just
+add `--cache-from type=local,mode=max,dest=<output folder>` to `dagger do build`.
+
+:::tip
+Using `mode=max` argument will cache **all** layers from intermediate
+steps, with is really useful in the context of Dagger where you will have
+multiple steps to execute.
+:::
+
+Here's an example that exports cache to `storage`.
+
+```shell
+dagger do build --cache-to type=local,mode=max,dest=storage 
+# ...
+
+tree storage -L 1   
+storage
+├── blobs
+├── index.json
+└── ingest
+```
+
+A new directory has been created that contains cache the run
+
+:::caution
+If you run different action in the plan, don't forget to store cache in different
+destination, so it will be not overwritten.
+:::
+
+To import the cache previously stored, you can use `--cache-from type=local,src=<cache folder>`.
+
+Here's an example, using a new buildkit daemon
+
+```shell
+# Down buildkit daemon
+docker container stop dagger-buildkitd && docker container rm dagger-buildkitd && docker volume rm dagger-buildkitd
+
+# Import cache on rebuild
+dagger do build --cache-to type=local,mode=max,dest=storage --cache-from type=local,src=storage
+[✔] actions.build.container                                                 2.3s
+[✔] client.filesystem.".".read                                              0.1s
+[✔] actions.build.container.export                                          0.0s
+[✔] client.filesystem."./output".write                                      0.4s
+```
+
+:::info
+In this part, we have how to keep cache in a local filesystem, if you want
+to see more options on local export, look at [Buildkit cache documentation](https://github.com/moby/buildkit#local-directory-1)
+:::
+
+## Persistent cache in a remote registry
+
+Buildkit can also import/export cache to a registry, it's a great way to share cache between your team and avoid
```

Comment:
```suggestion
Buildkit can also import/export cache to a registry. This is a great way to share cache between your team and avoid
```

Reply:
Fixed in https://github.com/dagger/dagger/pull/2035/commits/d916466adfe0a5cb53d851ba7e04033716be0afd